pages/osx/krunvm.md

```diff
@@ -0,0 +1,24 @@
+# krunvm
+
+> CLI-based utility for creating microVMs from OCI images.
+> More information: <https://github.com/containers/krunvm>.
+
+- Create MicroVM based on Fedora:
+
+`krunvm create docker.io/fedora --cpus {{number of vcpus}} --mem {{Memory in mbs}} --name {{name}}`
+
+- Start a MicroVM created:
+
+`krunvm start {{name}}`
+
+- List MicroVMs created:
+
+`krunvm list`
+
+- Change an already existant MicroVM:
+
+`krunvm changevm --cpus {{number of vcpus}} --mem {{memory in mbs}} --name {{new vm name}} {{current vm name}}`
+
+- Delete a microVM:
+
+`krunvm delete {{vm name}}`
```

Comment:
```suggestion
`krunvm delete "{{name}}"`
```

- Almost the same issue as above.
- Optional fix: consistency violation: placeholder can be renamed to `{{name}}` to be consistent with the first example (and others).